packages/big-design/src/components/FileUploader/utils/validateFile.ts

```diff
@@ -0,0 +1,31 @@
+import { ValidatorConfig } from '../FileUploader';
+
+export const validateFileFormat = (file: File | DataTransferItem, fileFormats = '') => {
+  const allowedFormats = fileFormats.split(',').map((format) => format.trim());
+  const fileExtension = file.type.split('/').pop()?.toLowerCase() ?? '';
+
+  return allowedFormats.some((format) => {
+    if (format === '*' || format === '.*') {
+      return true;
+    } else if (format.startsWith('.')) {
+      return format === `.${fileExtension}`;
+    } else if (
+      format.startsWith('audio/') ||
+      format.startsWith('video/') ||
+      format.startsWith('image/')
+    ) {
+      const mediaTypePrefix = format.split('/')[0];
+
+      return file.type.startsWith(`${mediaTypePrefix}/`);
+    }
+
+    return true;
+  });
+};
```

Comment:
```suggestion
export const validateFileFormat = (file: File | DataTransferItem, fileFormats = '') => {
  const allowedFormats = fileFormats.split(',').map((format) => format.trim());
  const fileExtension = file.type.split('/').pop()?.toLowerCase() ?? '';

  return allowedFormats.some((format) => {
    if (format.startsWith('.')) {
      return format === `.${fileExtension}`;
    }
    
    if (
      format.startsWith('audio/') ||
      format.startsWith('video/') ||
      format.startsWith('image/')
    ) {
      const mediaTypePrefix = format.split('/')[0];
      
      return file.type.startsWith(`${mediaTypePrefix}/`);
    }
    
    return true;
  });
};
```

Reply:
The suggestion brakes the the logic where the `fileFormats` equals to `.*`